docs/Installation.md

```diff
@@ -436,7 +436,7 @@ of operating system version and database engine version is supported.
 Configure Zonemaster::Backend to use the correct database engine:
 
 ```sh
-sed -i '/\bengine\b/ s/=.*/=MySQL/' /usr/local/etc/zonemaster/backend_config.ini
+sed -i '' '/[[:<:]]engine[[:>:]]/ s/=.*/=MySQL/' /usr/local/etc/zonemaster/backend_config.ini

```

Comment:
Instead of an meaningless `-i` use `-n` to surpress unneeded printout.

Reply:
The `-i` is not meaningless. It makes sed edit the file in-place, which is what's intended. Replacing the `-i ''` with `-n` would, on the other hand, make the command into a noop.

Reply:
You are right. For some reason the suffix must be set in FreeBSD `sed`. To make the command consistent between different OSs we should use Perl instead. Perl will always be available so it would not be a problem.